hack/test-end-to-end.sh

```diff
@@ -62,4 +63,8 @@ start_os_server
 # set our default KUBECONFIG location
 export KUBECONFIG="${ADMIN_KUBECONFIG}"
 
+os::test::junit::declare_suite_start "end-to-end/core"
```

Comment:
I'm not *particularly* interested in seeing what happens when there's a name collision in `junitreport` ... you mind deconflicting this name?